mautic/mautic
# Fix push contact ID to integration

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | No
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | ?



#### Description:



#### Steps to test this PR:

Push Contact ID to Salesforce field failed with this error: 

`Column not found: 1054 Unknown column 'l.mauticContactId' in 'field list`


<img width="505" alt="image" src="https://user-images.githubusercontent.com/462477/229330745-257b668c-bae3-4b60-abd3-56b5fe48d83f.png">



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Setup salesforce integration with Mautic 
3. Setup Contact ID to mapping
<img width="505" alt="image" src="https://user-images.githubusercontent.com/462477/229330791-4c86d34f-0642-4c52-bb61-7d88a66d36f7.png">
4. Push contact to SF
5.  You see error `Column not found: 1054 Unknown column 'l.mauticContactId' in 'field list`
6. After PR sync should work properly
